src/electron-forge-package.js

```diff
@@ -48,97 +48,35 @@ const main = async () => {
   const platform = program.platform || process.platform;
 
   const forgeConfig = await getForgeConfig(dir);
+  let packagerSpinner;
+
   const packageOpts = Object.assign({
     asar: false,
     overwrite: true,
   }, forgeConfig.electronPackagerConfig, {
     afterCopy: [async (buildPath, electronVersion, pPlatform, pArch, done) => {
       await pify(rimraf)(path.resolve(buildPath, 'node_modules/electron-compile/test'));
       done();
+    }, async (...args) => {
+      prepareSpinner.succeed();
+      await packagerCompileHook(dir, ...args);
+      packagerSpinner = ora.ora('Packaging Application').start();
     }].concat(forgeConfig.electronPackagerConfig.afterCopy ? forgeConfig.electronPackagerConfig.afterCopy.map(item => require(item)) : []),
```

Comment:
Hrm. I just realized this afterCopy code was here. Since it's different behavior than how Electron Packager works, it should be documented (in the README, for now).